### DISCUSSION TITLE
shields.io + operating-system-logos

### DISCUSSION THREAD

**Discussion by ghost** [badges/shields#8709] [Q&A] [answered]
Hi everyone.

#### concept
Shields does not support icons for os. So... there is a project called [EgoistDeveloper/operating-system-logos](https://github.com/EgoistDeveloper/operating-system-logos) that has icons for different operating systems, could I send a PR with these icons for shields.

#### input
`![Android](https://img.shields.io/badge/android?style=for-the-badge&logo=android&logoColor=gray)`

#### output
![image](https://user-images.githubusercontent.com/111701513/206983777-80ebe3ef-a862-49d7-b9d0-51ebfb88a041.png)

#### input
`![Windows](https://img.shields.io/badge/Windows?style=for-the-badge&logo=windows&logoColor=gray)`

#### output
![image](https://user-images.githubusercontent.com/111701513/206983981-81a9a65a-4cb3-4748-9ab7-4c1c7cb6253d.png)

#### alternative 3: [github/JunaidQadirB/font-os](https://github.com/JunaidQadirB/font-os)
*image*
![image](https://user-images.githubusercontent.com/111701513/208143168-f9d9725e-c206-4bc7-97a0-555529c742e9.png)

#### alternative 4: [github/fibo/os-icons8](https://github.com/fibo/os-icons8)
*image*
![image](https://user-images.githubusercontent.com/111701513/208142830-f99c5ae9-2922-486a-9536-bcb81f4e43eb.png)

*source-code*
```html 
<!--- https://github.com/fibo/os-icons8 --->
<img src="https://g14n.info/os-icons8/Apple-50.png" width="50" height="50" />
<img src="https://g14n.info/os-icons8/Linux-50.png" width="50" height="50" />
<img src="https://g14n.info/os-icons8/Free_BSD-50.png" width="50" height="50" />
<img src="https://g14n.info/os-icons8/Windows8-50.png" width="50" height="50" />
```

#### why?
1. I would like to unite two projects([shields.io](https://shields.io/) + [EgoistDeveloper/operating-system-logos](https://github.com/EgoistDeveloper/operating-system-logos)) into something unique.
2. I wonder if the shields project could provide logos for all new, old operating systems. 
3. In addition to these reasons, this allows developers to demonstrate what software support is done in README.md files.
4. A good use case would be distrochooser, which is an online tool to help choose the linux distro, there might be a need to use shields with os for this.
5. I can add metadata in my open projects, if my created softwares are compatible with some linux distribution or operating system
6. Shields does not support icons for os. So... there is a project called EgoistDeveloper/operating-system-logos that has icons for different operating systems, could I send a PR with these icons for shields.

#### question/feedback
*What do you all think about this idea?*

**Comment by relliv** [comment, marked as answer]
Hi, I am the maintainer of [EgoistDeveloper/operating-system-logos](https://github.com/EgoistDeveloper/operating-system-logos) and I created this repo for my statistic tables or reports. And community uses with similar projects but I didn't think it with shields.io but it's an awesome idea.

Pros: This set fits badges and there are 80+ logos in different sizes.
Cons: There is no SVG and file names must be mapped to specific usages.

So, I can vote for my project and I am ready to help.

**Reply by ghost** [reply]
Hi EgoistDeveloper.

> Pros: This set fits badges and there are 80+ logos in different sizes.
> Cons: There is no SVG and file names must be mapped to specific usages.

thank you for feedback.

> So, I can vote for my project and I am ready to help.

great idea.
